### ISSUE TITLE
Heart Rate is not syncing

### ISSUE THREAD

**Issue by WindRider7** [praveendath92/fitbit-googlefit#34]
Hello ! I tried to check this graph https://www.reddit.com/r/AndroidWear/comments/5rmt3b/how_to_see_google_fit_heart_rate_history_on_phone/ But selecting hearrate does not display anything, can this app feed google fit with heart rate data from fitbit ?

**Comment by praveendath92** [comment]
@WindRider7 Yes, it should be able to. Something seems to be broken in Google Fit APIs. Will take a look.

**Comment by praveendath92** [comment]
I just ran a sync for the last 3 days. Everything synced perfectly. I could see all my steps - perfectly matching with Fitbit, running activities and heart rate too.

**Comment by praveendath92** [comment]
<img width="703" alt="screen shot 2017-06-26 at 6 05 19 am" src="https://user-images.githubusercontent.com/816666/27525403-7dc4c386-5a35-11e7-8823-e4634d92f436.png">

**Comment by praveendath92** [comment]
@WindRider7 I tried to check Heart rate graph on my phone and I couldn't find it either. That doesn't mean it's not synced by the app. Because, you can see average heart rate calculated for a run.

**Comment by WindRider7** [comment]
You are right, I can see average heart rate for run also.

**Comment by unicorn855** [comment]
@WindRider7 @praveendath92 I have had this happen for me as well, the graphs for pulse are oddly averaged it seems (maybe each of the bars represents about an hour? or maybe 40ish minutes? certainly not a very user friendly layout which Google is offering there so far :(  ) but then again, for the activities, the correct average bpm is displayed (however, it's not stored correctly on Google's side it seems, it's really quite an odd thing)
